src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -360,4 +360,7 @@
   <data name="PX1071Title" xml:space="preserve">
     <value>Actions cannot be executed within event handlers</value>
   </data>
+  <data name="PX1080Title" xml:space="preserve">
+    <value>A data view delegate should not start PXLongOperation</value>
```

Comment:
Is this string verified?
Please see the corresponding string for event handlers - it uses different terms:

> A long-running operation cannot be started within an event handler

Reply:
No its not. I've requested Kseniya to review the string in Jira when I've created this pull-request.
Now the string is reviewed and I'll change it.